Makefile

```diff
@@ -21,16 +23,30 @@ golangci-lint: ## Run golangci-lint on codebase
 	@echo Running golangci-lint
 	golangci-lint run ./...
 
+.PHONY: test
 test:
 	go test -v -mod=readonly -failfast -race ./...
 
+.PHONY: build
 build:
 	mkdir -p dist
 	env GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o dist/rtcd -ldflags '$(LDFLAGS)' -mod=readonly -trimpath ./cmd/rtcd
 
+.PHONY: docker-build
+docker-build:
+	@if ! [ -x "$$(command -v docker)" ]; then \
+		echo "Docker is not installed. Please see https://docs.docker.com/get-docker for installation instructions."; \
+		exit 127; \
+	fi; \
+
+	@[ "${tag}" ] || ( echo "build-docker requires a tag. Set a tag with: \"make docker-build tag=rtcd:my_tag\""; exit 1 )
```

Comment:
Is it possible to exit with another code. It is easy to confuse exit code `1` with general errors. Suggestion `128`- `Invalid Argument`

Reply:
Actually `127` is `command not found`, which is exactly the case here: 
http://www.bic.mni.mcgill.ca/~dale/helppages/BashGuide/advshell/exitcodes.html

Reply:
disregard my comment, I see you are commenting  online 42

Reply:
Rebased and pushed changes requested @pfltdv